frontend/src/components/ProjectList/components/Dataset/components/DownloadDataset/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from "react";
```

Comment:
The structure of `<DownloadDataset />` (exhibit 2) should be similar to its peer `<ViewDataset />`

src: `frontend/src/components/ProjectList/components/Dataset/components/ViewDataset/index.tsx`

Which contains a `<Wrapper />` component for styling the column

We also need to add a new column header "Download dataset" (exhibit 1) in `frontend/src/components/ProjectList/components/Heading/index.tsx`

```tsx
const Heading: FC = () => {
  return (
    <Wrapper>
      <Name>Dataset name</Name>
      <View>
        View in cellxgene
        <span title={TOOLTIP_MESSAGE}>
          <QuestionMark src={questionMarkSvg} alt="question mark" />
        </span>
      </View>
      <Download /> <---------------- NEW
      <Info>More information</Info>
    </Wrapper>
  );
};
```

<img width="1187" alt="Screen Shot 2020-09-11 at 12 38 56 PM" src="https://user-images.githubusercontent.com/6309723/92966149-d9055d80-f42b-11ea-9038-6220daee5820.png">

Reply:
@Bento007 PTAL 😄 Thank you!